### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Ade179**
Project has been completed 

- Followed Gitflow
- Project features are in the Readme.md file.
Below is a link to the loom video for the description of the project
https://www.loom.com/share/b2168422439f43d1a792b6787bad3380

### CODE REVIEW

File: src/Components/nav.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+.header {
+  background-color: #a50e63;
+  padding: 2rem;
+  color: aliceblue;
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+}
+
```
Comment (sja-thedude) [inline src/Components/nav.css:14]: - [ ] Kindly make sure that your app design follows the design guidelines [here](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)).

File: src/test/Navigation.test.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Nav from '../Components/Navigation';
+
+test('should match the snapshot', () => {
+  const home = renderer.create(<Nav />);
```
Comment (sja-thedude) [inline src/test/Navigation.test.js:6]: - [ ] Kindly fix the error, currently one is failing.

Screenshot 2022-12-17 at 1 47 11 AM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Ade179, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ All linters passing
✔️ Professional Readme
✔️ Good presentation
✔️ Correct Deployment of the App

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mwanawabangona — APPROVED** [review]
Hi @Ade179,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**